Sources/SwiftBuildSupport/PIF.swift

```diff
@@ -27,39 +29,40 @@ import struct TSCBasic.ByteString
 /// between builds which use different schemes or configurations), and can be
 /// incrementally updated by clients when something changes.
 public enum PIF {
-    /// This is used as part of the signature for the high-level PIF objects, to ensure that changes to the PIF schema
-    /// are represented by the objects which do not use a content-based signature scheme (workspaces and projects,
-    /// currently).
-    static let schemaVersion = 11
```

Comment:
Both workspaces and projects use a content-based signature now. See this comment.